### DISCUSSION TITLE
can we get sslinspection rule / insight logs / cloud app control rule via api ?

### DISCUSSION THREAD

**Discussion by lishuangqing01** [mitchos/pyZscaler#222] [Q&A]
Can we get SSL inspection rule / Web<Firewall> insight logs / Cloud app control rule via API  or Pyzscaler ?

**Comment by mitchos** [comment]
Hi @lishuangqing01 Zscaler has not yet added public API endpoints for SSL Inspection, Web Insight Logs or Cloud App Control. 

I normally keep an eye on new API endpoints and then create an issue to track them. I'll update this thread and ping you if/when they expose these ones. 

Definitely a lot of people hanging out for the release of some of these, myself included. 

If you're a Zscaler customer, I'd also encourage you to reach out to your account team and ask to be added to any enhancement requests (ER) that track these features.
